src/test/java/uk/gov/pay/publicauth/it/PublicAuthResourceIT.java

```diff
@@ -66,22 +67,22 @@ class PublicAuthResourceIT {
     private static final String TOKEN_HASH_COLUMN = "token_hash";
     private static final String CREATED_USER_NAME = "user-name";
     private static final String CREATED_USER_NAME2 = "user-name-2";
-    private final String validTokenPayload = new Gson().toJson(
+    private final String validTokenPayload = gson.toJson(
```

Comment:
I'm not gonna insist on it but as you pointed out we could do away with the JSON converter altogether right?

Reply:
Think I'll leave this to another PR.